guides/source/active_job_basics.md

```diff
@@ -123,6 +150,11 @@ module YourApp
 end
 ```
 
+NOTE: Since jobs run in parallel to your Rails application, most queuing implementations
+require that you start an implementation-specific queuing service (in addition to
+starting your Rails app) for the job processing to work. For information on
+how to do that refer to the documentation of your respective queuing implementation.
```

Comment:
I realized that the use of enqueueing/enqueuing has already been inconsistent throughout the document. I could normalize the other uses, too. I propose "enqueuing" as the stanard.